### DISCUSSION TITLE
No Google popup on second connection (auto connection with same account)

### DISCUSSION THREAD

**Discussion by kapik** [capawesome-team/capacitor-firebase#493] [Q&A]
Hello everyone !!

I have a problem when connecting with Google.
When I log in twice in a row, the second time the Google popup does not appear and I log in with the previous google account.
In my complete process I send the credential to AngularFire to log in with.

When I log out, I do it on both accounts: 
```ts
    await FirebaseAuthentication.signOut()
    await this.auth.signOut() // AngularFire
```

I guess it's a disconnection problem, but I don't understand.

Do you have any clues?
Thanks for your help

**Comment by robingenz** [comment]
Thank you for your request! This seems to be a bug. Please create a bug report: https://github.com/capawesome-team/capacitor-firebase/issues/new/choose
